addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2015,6 +2015,17 @@ options.registry.SnippetMove = options.Class.extend({
                 }
                 break;
         }
+        $('html, body').animate(
```

Comment:
Commit message: no uppercase letter for the title + as the PR contains multiple commit, need a mention of the PR in each commit "Part of <pr-link>".

Reply:
Yep!